core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChildSetterDescriptor.java

```diff
@@ -0,0 +1,35 @@
+package nl.nn.adapterframework.doc.model;
+
+import org.xml.sax.SAXException;
+
+import lombok.Getter;
+
+/**
+ * This class is similar to {@link ConfigChild}, but it is not the same. As an example,
```

Comment:
Ik denk dat je deze javadoc het beste kan beginnen met wat het is, bijvoorbeeld:
This class describes a method that a child element can call to register itself at its parent.
If such a method exists for a class, that class can have ConfigChildren of the type of the parameter of the method.

Reply:
Akkoord, ga ik verbeteren.

Reply:
Ik heb de Javadoc veranderd, maar er zit nog iets niet lekker. Ik vermoed dat ik altijd een ander idee heb gehad dan jij over wat ConfigChildSetterDescriptor is. Ik ben benieuwd wat je van de Javadoc vindt die ik gemaakt heb. Is de naam ConfigChildSetterDescriptor nu nog goed? Of zal ik je suggestie over de Javadoc tekst overnemen zoals je die gegeven hebt?